apps/src/studio/collidable.js

```diff
@@ -7,7 +7,7 @@
 
 'use strict';
 
-var Studio = require('./studio');
+var studioApp = require('../StudioApp').singleton;
```

Comment:
This is a workaround for a circular dependency related issue that was causing the [factor-bundle](https://github.com/substack/factor-bundle) plugin to bundle most of the Studio app into the `common.js` file. Removing the reference to `./studio` within this file seemed to fix the issue for now.